src/infection-model/seir.tsx

```diff
@@ -307,31 +315,62 @@ export function getAllBracketCurves(inputs: CurveProjectionInputs) {
   ).forEach(([pop, cases, recovered, deaths], index) => {
     // distribute cases across compartments proportionally
 
-    const infectious = cases * pInitiallyInfectious;
-    // exposed is related to the number of infectious
-    // but it can't be more than the total uninfected population
-    const exposed = Math.min(infectious * ratioExposedToInfected, pop - cases);
-
-    singleDayState.set(index, seirIndex.susceptible, pop - cases - exposed);
-    singleDayState.set(index, seirIndex.exposed, exposed);
-    singleDayState.set(index, seirIndex.infectious, infectious);
-    singleDayState.set(index, seirIndex.mild, cases * pInitiallyMild);
-    singleDayState.set(index, seirIndex.severe, cases * pInitiallySevere);
-    singleDayState.set(
-      index,
-      seirIndex.hospitalized,
-      cases * pInitiallyHospitalized,
-    );
-    singleDayState.set(
-      index,
-      seirIndex.mildRecovered,
-      cases * pInitiallyMildRecovered,
-    );
-    singleDayState.set(
-      index,
-      seirIndex.severeRecovered,
-      cases * pInitiallySevereRecovered,
-    );
+    if ((typeof recovered !== 'undefined') && (typeof recovered !== 'undefined')) {
```

Comment:
was one of these `recovered`s supposed to be `deaths`?

Reply:
Yes! Just updated that, thanks for catching it